.buildkite/pipeline-deploy.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -euo pipefail
+
+cat <<EOF
+env:
+  BUILDKITE_ARTIFACT_UPLOAD_DESTINATION: s3://vivi-devops-packages/gstreamer/
+  BUILDKITE_S3_DEFAULT_REGION: ap-southeast-2
+
+steps:
+  - label: ":s3: Deploy"
+    depends_on:
+      - package
+    agents:
+      queue: v3
+    command: .buildkite/scripts/links.sh ${VIVI_FILENAME}
+    plugins:
+      - artifacts#309df16:
```

Comment:
I'm sure I asked before but what's this particular artifacts plugin commit has, can't we use 1.2.0 or whatever the latest is?

Reply:
they haven't tagged the changes which allow renaming artifacts